src/components/common/buttons/ButtonWithDoneState.js

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { StyleSheet, View } from 'react-native'
+import Icon from '../view/Icon'
+import CustomButton from './CustomButton'
+
+const NOT_EXECUTED = 'NOT_EXECUTED'
```

Comment:
probably better to define those constants somewhere in transactions modules ? e.g. in the wallet ?

Reply:
this const needed and used only in the scope of this file, so no need to move them anywhere.